src/core/manager.c

```diff
@@ -4350,16 +4351,31 @@ int manager_dispatch_user_lookup_fd(sd_event_source *source, int fd, uint32_t re
         return 0;
 }
 
+static int short_uid_range(const char *path) {
+        _cleanup_free_ UidRange *p = NULL;
+        size_t n = 0;
+        int r;
+
+        assert(path);
+
+        r = uid_range_load_userns(&p, &n, NULL);
```

Comment:
`NULL` -> `path`??

Otherwise, `path` is used only in the log message. And gid map will never checked.

Reply:
ouch